pandas/tests/indexes/common.py

```diff
@@ -404,7 +404,7 @@ def test_difference_base(self):
                 with tm.assertRaisesRegexp(TypeError, msg):
                     result = first.difference([1, 2, 3])
 
-    def test_symmetric_diff(self):
+    def test_symmetric_difference(self):
```

Comment:
IIRC there is a test for these types of deprecations, pls add one for ``symmetric_difference``